packages/core/src/render3/instructions/element.ts

```diff
@@ -54,9 +54,14 @@ function elementStartFirstCreatePass(
   const attrs = getConstant<TAttributes>(tViewConsts, attrsIndex);
   const tNode = getOrCreateTNode(tView, index, TNodeType.Element, name, attrs);
 
-  const hasDirectives =
-      resolveDirectives(tView, lView, tNode, getConstant<string[]>(tViewConsts, localRefsIndex));
-  ngDevMode && validateElementIsKnown(native, tNode.value, tView.schemas, hasDirectives);
+  if (ngDevMode) {
+    const hasDirectives =
+        resolveDirectives(tView, lView, tNode, getConstant<string[]>(tViewConsts, localRefsIndex));
+
+    const hostIsStandalone = isHostComponentStandalone(lView);
+
+    validateElementIsKnown(native, tNode.value, tView.schemas, hasDirectives, hostIsStandalone);
+  }
```

Comment:
super nit: I think we can remove new lines in between:
```suggestion
    const hasDirectives =
        resolveDirectives(tView, lView, tNode, getConstant<string[]>(tViewConsts, localRefsIndex));
    const hostIsStandalone = isHostComponentStandalone(lView);
    validateElementIsKnown(native, tNode.value, tView.schemas, hasDirectives, hostIsStandalone);
  }
```
Usually I use new lines to separate a particular block of code from the rest of the code, but in our case this is a single logical block, so we can just remove line breaks.